client/main.js

```diff
@@ -100,6 +106,58 @@ $(document).ready(function () {
         page.updateVolIcon(audioElement);
     });
 
+    // How I imagine this would work vvv, the console logs didn't even work
+
+    $('#pop-settings').click(function () {
+        // console.log("in pop settings");
+        // console.log(settingsDisplay[0].style["display"]);
+        let shown = settingsDisplay[0].style["display"];
+        if (shown == "none") {
+            settingsDisplay[0].style["display"] = "block";
+        } else {
+            settingsDisplay[0].style["display"] = "none";
+        }  
+    });
+
+    $('#start-timer').click(function () {
+
+        let shown = startDisplay[0].style["display"];
+
+        if (shown == "none") {
+            startDisplay[0].style["display"] = "inline-block";
+            stopDisplay[0].style["display"] = "none";
+        } else {
+            startDisplay[0].style["display"] = "none";
+            stopDisplay[0].style["display"] = "inline-block";
+        }  
+    });
+
+
+    $('#stop-timer').click(function () {
+
+        let shown = startDisplay[0].style["display"];
+
+        if (shown == "none") {
+            startDisplay[0].style["display"] = "inline-block";
+            stopDisplay[0].style["display"] = "none";
+        } else {
+            startDisplay[0].style["display"] = "none";
+            stopDisplay[0].style["display"] = "inline-block";
+        }  
+    });
+
+    $('#vol-change').click(function () {
+        
+        let shown = volumeDisplay[0].style["display"];
+        if (shown == "none") {
+            volumeDisplay[0].style["display"] = "inline-block";
+            titleDisplay[0].style["display"] = "none";
+        } else {
+            volumeDisplay[0].style["display"] = "none";
+            titleDisplay[0].style["display"] = "inline-block";
+        }  
+    });
+
     //Updates volume when slider is changed.
     $('#vol-control').on("input change", function () {
         tempVol = this.value;
```

Comment:
(not related to `main.js`, I just can't leave a comment on a .png) Looks like `/public/images/officebeatz-banner copy.png` is a copy mistake, delete it?

Reply:
removed ty!